drivers/hud_spm_report/app/models/hud_spm_report/fy2024/return.rb

```diff
@@ -18,8 +18,9 @@ class Return < HudReports::ReportClientBase
 
     has_many :hud_reports_universe_members, inverse_of: :universe_membership, class_name: 'HudReports::UniverseMember', foreign_key: :universe_membership_id
 
+    # duck-types to enrollment
     def project_id
```

Comment:
This was returning the project_id string, not the project pk. 

I reviewed the current SPM module. It seems like the return.project_id is only used here so I have to imagine this method is incorrect. Note, it would technically be safer to make a new project_pk method instead of changing this behavior but I'm choosing to fix it here in the name of maintainability.